_posts/2020-01-01-2019-summary.md

```diff
@@ -0,0 +1,230 @@
+  mounting in configuration and can be used for Kubernetes deployments.
+* LZ4/ZSTD support for Parquet and ORC. These are the recommended compression algorithms.
+* Support for insert-only Hive transactional tables and Hive bucketing v2 as part of 
+  [making Presto compatible with Hive 3]({% post_url 2019-12-28-hive-3 %}).
+* Improvements in ANALYZE statement for Hive connector.
+* Support for [multiple files per bucket]({% post_url 2019-05-29-improved-hive-bucketing %}) 
+  for Hive tables. This allows inserting data into bucketed tables without having to rewrite entire partitions.
+* Support for upper- and mixed-case table and column names in JDBC-based connectors.
+* New features and improvements in type mappings in PostgreSQL, MySQL, SQL Server and Redshift
+  connectors. This includes support for PostgreSQL arrays and `timestamp with time zone` type, and 
+  the ability to read columns of unsupported types.
+* Improvements in [Hive compatibility with Hive version 2.3](https://github.com/prestosql/presto/pull/833) 
+  and [with CDH's Hive](https://github.com/prestosql/presto/pull/1937).
+* Lots and lots of bug fixes!
+
+# Coming Up...
+
+These are some of the projects that are currently in progress and are likely to land in the short term.
+
+* Support for pushing down row dereference expressions into connectors. This will help reduce 
+  the amount of data and CPU needed to process highly nested columnar formats such as ORC and Parquet.
+* Extend dynamic filtering to support distributed joins and other operators. Use dynamic filters for 
+  pruning partitions at runtime when querying Hive.
+* [Extended late materialization](https://github.com/prestosql/presto/pull/2418) support to queries involving 
+  complex correlated subqueries.
+* Finalize [Hive 3 support]({% post_url 2019-12-28-hive-3 %}).
+* Improved [INSERT into partitioned tables](https://github.com/prestosql/presto/pull/2358), which will help with 
+  large ETL queries.
+* [Improvements and features](https://github.com/prestosql/presto/issues/1324) in Iceberg connector.
+* [Pinot](https://github.com/prestosql/presto/pull/2028) connector.
+* [Oracle](https://github.com/prestosql/presto/pull/1959) connector.
+* [Influx](https://github.com/prestosql/presto/pull/2397) connector.
+* [Prometheus](https://github.com/prestosql/presto/pull/2321) connector.
+* [Salesforce](https://prestosql.slack.com/archives/CQT2JH4KG/p1576038838027500) connector. 
+* Support for [Confluent registry in Kafka connector](https://github.com/prestosql/presto/pull/2106).
+* Revamp of the function registry and function resolution to support dynamically-resolved 
+  functions and SQL-defined functions.
+* A new [Parquet writer](https://github.com/prestosql/presto/pull/2004) optimized to work efficiently 
+  within Presto.
+
+... and many, many more. Follow us on [Twitter](https://twitter.com/prestosql) and 
+[join our Slack channel](https://prestosql.io/slack.html).
```

Comment:
Remove "channel" since we are asking them to join our Slack instance (not sure what to call it but "channel" is wrong)

Reply:
in slack parlance this is called "workspace", but i dont think we should refer to it as this.
+1 for "Slack" alone